.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+### Prerequisites
+
+- [ ] Have you written a descriptive issue title?

```

Comment:
I'd like to see these as `I have...`. This is just a santiy-check for the author, right?